src/github/client/github-client.ts

```diff
@@ -112,8 +112,9 @@ export default class GitHubClient {
 	/**
 	 * Get a single pull request for the given repository.
 	 */
+	// TODO: add a unit test
 	public async getPullRequest(owner: string, repo: string, pullNumber: string): Promise<AxiosResponse<Octokit.PullsGetResponse>> {
-		return await this.get<Octokit.PullsGetResponse>(`/repos/:owner/:repo/pulls/:pullNumber`, {
+		return await this.get<Octokit.PullsGetResponse>(`/repos/:owner/:repo/pulls/:pullNumber`, {},  {
```

Comment:
context: we added `{}` as the second parameter here, because we forgot to do that in the new `getCommit()` function that we added and got an error in the test. So we added it in these two functions, as well, but we skipped writing a test for those functions for now (there wasn't a test, yet)